DEVELOPER.md

```diff
@@ -67,36 +67,60 @@ The `workspace/index.js` file is then loaded into `workspace/index.html` via `<s
 
 ##### Process
 
-1. Define the widget's template. The base widgets templates list is retrieved from the MNO API, within each dashboard object. Each widget template will have a structure similar to the following:
+1. **Defining the Widgets Template.**<br>
+  *Widgets templates are currently kept in the maestrano api. They declare defining attributes for each widget. *
+  *It is important to take not of the `path` vs `path` & `metadata.template` attributes. Defining a `metadata.template` enables you to use an existing Impac API engine, and point the front-end to a different template*
```

Comment:
typos:
- "not**e**"
- point**s**

Is it mentioned somewhere that by default, "path" will correspond to both the engine path and the template name?